### PR TITLE
Touchdown Update for Localization strings

### PR DESCRIPTION
**Pull request by anjulgarg**
# What
Adds localization for missing strings.

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by JamesBurnside — APPROVED** [review]
Woohoo! (Are the automated PRs for this not working currently?)

**Review by alkwa-msft — APPROVED** [review]

**Comment by anjulgarg** [comment]
> Woohoo! (Are the automated PRs for this not working currently?)

I didn't know we  had automated PR setup for this.
@mgamis-msft might know more.

File: packages/react-components/src/localization/locales/fr-FR/strings.json [hunk 1]
```diff
@@ -2,7 +2,9 @@
   "participantItem": {
     "isMeText": "(vous)",
     "menuTitle": "Plus d’options",
-    "removeButtonLabel": "Supprimer"
+    "removeButtonLabel": "Supprimer",
+    "sharingIconLabel": "Partage",
+    "mutedIconLabel": "Muet activé"
```
Comment (alcail) [inline packages/react-components/src/localization/locales/fr-FR/strings.json:7]: why does it sound so wrong :D
Might be me not talking French so often or at least not using French language in software/browser...

File: packages/react-components/src/localization/locales/fr-FR/strings.json [hunk 2]
```diff
@@ -76,6 +89,21 @@
...
     "startScreenShareGeneric": "Échec du démarrage du partage d’écran",
-    "stopScreenShareGeneric": "Échec de l’arrêt du partage d’écran"
+    "stopScreenShareGeneric": "Échec de l’arrêt du partage d’écran",
+    "callNetworkQualityLow": "La qualité de réseau est faible.",
+    "callNoSpeakerFound": "Aucun haut-parleur ou casque n’a été trouvé. Connectez un périphérique audio pour entendre l’appel.",
+    "callNoMicrophoneFound": "Aucun micro n’a été trouvé. Connectez un périphérique d’entrée audio.",
+    "callMicrophoneAccessDenied": "Impossible d’accéder au microphone. Cliquez sur le verrou dans la barre d’adresses pour accorder l’autorisation à cette page Web.",
+    "callMicrophoneMutedBySystem": "Votre micro est désactivé par votre système.",
+    "callMacOsMicrophoneAccessDenied": "Impossible d’accéder au microphone. Accordez l’autorisation de microphone dans vos paramètres de confidentialité macOS.",
+    "callLocalVideoFreeze": "La bande passante réseau est médiocre. Votre vidéo peut apparaître suspendue pour les autres participants à l’appel.",
```
Comment (alcail) [inline packages/react-components/src/localization/locales/fr-FR/strings.json:99]: lol: 'suspendue" (suspended/hanged)... "figée" (frozen) would be a better fit here

File: packages/react-components/src/localization/locales/fr-FR/strings.json [hunk 3]
```diff
@@ -76,6 +89,21 @@
...
+    "callCameraAccessDenied": "Impossible d’accéder à la caméra. Cliquez sur le verrou dans la barre d’adresses pour accorder l’autorisation à cette page Web.",
+    "callCameraAlreadyInUse": "Impossible d’accéder à la caméra. Il est peut-être déjà utilisé par une autre application.",
```
Comment (alcail) [inline packages/react-components/src/localization/locales/fr-FR/strings.json:101]: OMG, "Il est peut-être déjà utilisé ..." Caméra is female, so "**Elle** est peut-être déjà **utilisée** ... "

**Comment by mgamis-msft** [comment]
> Woohoo! (Are the automated PRs for this not working currently?)

No, automated PRs are not working currently. I will work on creating a machine account for azure pipelines that has push permissions to create a PR and with that create weekly azure pipeline to commit to a non-main branch (which is tricky because I have to somehow branch off the trigger branch which is main).

File: packages/react-components/src/localization/locales/fr-FR/strings.json [hunk 4]
```diff
@@ -61,7 +64,17 @@
     "friday": "Vendredi",
     "saturday": "Samedi",
     "participantJoined": "a rejoint la conversation.",
-    "participantLeft": "a quitté le conversation."
+    "participantLeft": "a quitté le conversation.",
```
Comment (rceraline) [inline packages/react-components/src/localization/locales/fr-FR/strings.json:67]: Should be "a quitté **la** conversation."

Comment (alcail) [reply packages/react-components/src/localization/locales/fr-FR/strings.json:67]: yep, missed this one :)

**Pull request merged**